assets/js/components.js

```diff
@@ -47,6 +47,30 @@ jQuery(function () {
         $(td).html(`${rowData.attributes.github_enforce_admins_enabled}`)
       },
     },
+    {
+      targets: 5,
```

Comment:
Shouldn't be needed as you're using `columns` and this option is for `columnDefs` 🙂